core/types/rlp_backwards_compat.libevm_test.go

```diff
@@ -138,34 +142,48 @@ func TestBodyRLPBackwardsCompatibility(t *testing.T) {
 	for _, tx := range txMatrix {
 		for _, u := range uncleMatrix {
 			for _, w := range withdrawMatrix {
-				bodies = append(bodies, &Body{tx, u, w})
+				body := &Body{
+					Transactions: tx,
+					Uncles:       u,
+					Withdrawals:  w,
+				}
```

Comment:
Change to (in order to make sure all fields are set):

```suggestion
				body := &Body{tx, u, w, nil /* extras field */})
```